### PR TITLE
[DE-602] test CRUD for subcodes

### PR DESCRIPTION
**Pull request by jumper2210**

### CODE REVIEW

File: e2e/src/couponsController.spec.ts [hunk 1]
```diff
@@ -36,6 +36,33 @@ const couponBody = {
   },
 };
 
+const createSubdodes = async (
```
Comment (mikepawlinski) [inline e2e/src/couponsController.spec.ts:39]: typo

File: e2e/src/couponsController.spec.ts [hunk 2]
```diff
@@ -36,6 +36,33 @@ const couponBody = {
...
+  code: string,
+  subcodes: string[],
+  couponsController: CouponsController,
+  productFamilyId: number
+) => {
+  const createReponse = await couponsController.createCoupon(productFamilyId, {
+    coupon: {
+      ...couponBody.coupon,
+      code,
+      productFamilyId: String(productFamilyId),
```
Comment (mikepawlinski) [inline e2e/src/couponsController.spec.ts:49]: use `toString` instead

File: e2e/src/couponsController.spec.ts (hunk 1 above)
Comment (jumper2210) [reply e2e/src/couponsController.spec.ts]: done 👍

File: e2e/src/couponsController.spec.ts (hunk 2 above)
Comment (jumper2210) [reply e2e/src/couponsController.spec.ts]: done 👍

**Review by mikepawlinski — APPROVED** [review]

**Pull request merged**